fusor-ember-cli/app/controllers/register-nodes.js

```diff
@@ -645,14 +664,19 @@ export default Ember.Controller.extend(ProgressBarMixin, NeedsDeploymentMixin, {
   updateAutoDetectedNodes(hostArray) {
     let autoDetectedNodesMultiMac = [];
     let autoDetectedNodesSingleMac = [];
+    let usedMacs = this.getPortMacAddresses();
 
-    hostArray.forEach((hostHash) => {
+    hostArray.forEach(hostHash => {
```

Comment:
I've still been wrapping single arg fat arrows in paren. Don't feel strongly either way, is your preferred style no paren? Just want to make sure we're consistent going forwards.